File: JAVASCRIPT PROGRAMMING/Test43.js
"use strict";
// 43. FIBONACCI SERIES
const fibonacci = (range) => {
    let num0 = 0;
    let num1 = 1;
    let temp = 0;
    process.stdout.write(`${num0} ${num1} `);
    for (let i = 3; i <= range; i++) {
        temp = num0 + num1;
        num0 = num1;
        num1 = temp;
        process.stdout.write(`${temp} `);
    }
    console.log();
};
fibonacci(10);
fibonacci(20);

File: JAVA PROGRAMMING/Test43.java
// 43. FIBONACCI SERIES
import java.util.Scanner;
public class Test43 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        int num0 = 0, num1 = 1, temp = 0;
        System.out.print(num0+" "+num1+" ");
        for(int i = 3; i <= range; i++) {
            temp = num0 + num1;
            num0 = num1;
            num1 = temp;
            System.out.print(temp+" ");
        }
        in.close();
    }
}